libs/sdk-ui-geo/src/GeoPushpinChart.tsx

```diff
@@ -92,7 +92,7 @@ const NON_CORE_PROPS: Array<keyof IGeoPushpinChartProps> = [
 ];
 
 function GeoPushpinChartInner(props: IGeoPushpinChartProps): JSX.Element {
-    const { backend, workspace, sortBy, filters, exportTitle } = props;
+    const { backend, workspace, sortBy, filters, exportTitle, execConfig = {} } = props;
```

Comment:
The dafault to {} can cause problems, please remove it  (similarly to the other charts).

Reply:
unfortunately linting complains about it being undefined.

Reply:
Oh, right, there is different strict setting. Ok then